src/___tests___/htpassed.test.js

```diff
@@ -76,4 +76,35 @@ describe('HTPasswd', () => {
     };
     wrapper.adduser('usernotpresent', 'somerandompassword', callback);
   });
+
+  it('changePassword - it should throw an error', done => {
+    const callback = (a, b) => {
```

Comment:
`a, b` is not a good name, I'd chose something like 3 months later reminder what means `a` or `b`.

Reply:
The same below this file